### PR TITLE
CCDB Landing Page Map Updates

### PR DESCRIPTION
**Pull request by sephcoster**
Updates to the CCDB landing page and bumps to CCDB search application libraries to include a map, print views, updated accessibility and responsive design elements, and revised language.

EDIT: Please do not merge / deploy at this time.

## Additions

- Map features on landing page and Search page
- Updated print views
- Javascript unprocessed code to support landing page maps on legacy template page

## Removals

- None

## Changes

- CCDB-UI and CCDB-API dependency updates
- Language changes and updates

## Testing

1. 💯 

## Checklist

- [x] PR has an informative and human-readable title
- [x] Changes are limited to a single goal (no scope creep)
- [x] Code can be automatically merged (no conflicts)
- [x] Code follows the standards laid out in the [CFPB development guidelines](https://github.com/cfpb/development)
- [x] Passes all existing automated tests
- [x] Any _change_ in functionality is tested
- [x] New functions are documented (with a description, list of inputs, and expected output)
- [ ] Placeholder code is flagged / future todos are captured in comments
- [x] Visually tested in supported browsers and devices (see checklist below :point_down:)
- [ ] Project documentation has been updated
- [x] Reviewers requested with the [Reviewers tool](https://help.github.com/articles/requesting-a-pull-request-review/) :arrow_right:

### Browsers

- [ ] Chrome on desktop
- [ ] Firefox
- [ ] Safari on macOS
- [ ] Edge
- [ ] Internet Explorer 9, 10, and 11
- [ ] Safari on iOS
- [ ] Chrome on Android

### Accessibility

- [ ] Keyboard friendly
- [ ] Screen reader friendly

### Other

- [ ] Is useable without CSS
- [ ] Is useable without JS
- [x] Flexible from small to large screens
- [ ] No linting errors or warnings
- [ ] JavaScript tests are passing

### CODE REVIEW

File: cfgov/unprocessed/apps/ccdb-landing-map/package.json [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+    "type": "git",
+    "url": "https://github.com/cfpb/cfgov-refresh.git"
+  },
+  "license": "SEE LICENSE IN TERMS.md",
+  "engines": {
+    "node": ">=8.0.0"
+  },
+  "dependencies": {
+    "@cfpb/cfpb-core": "0.0.9",
+    "@cfpb/cfpb-layout": "0.0.9",
```
Comment (Scotchester) [inline cfgov/unprocessed/apps/ccdb-landing-map/package.json:20]: These should be removable, as they are available in the wider project already. I removed them, reran `frontend.sh`, and everything still works for me locally.
```suggestion
```

File: cfgov/legacy/templates/complaint/complaint-landing.html [hunk 2]
```diff
@@ -5,8 +5,7 @@
 
 {% block page_css %}
 <!-- The CSS file below was generated using LESS files from github.com/cfpb/complaint -->
```
[the remark this answers is not in the document]
Comment (sephcoster) [reply cfgov/legacy/templates/complaint/complaint-landing.html:7]: To help future "us" remember where this came from (and how it was built) we'd like to keep it in there until we deprecate the legacy CSS in favor of an updated template if comments are OK?

File: cfgov/unprocessed/apps/ccdb-landing-map/js/TileMap.js [hunk 3]
```diff
@@ -0,0 +1,468 @@
...
+ * @param {object} t the highchart element containing point, and other props
+ */
+export function clickHandler( isPerCapita, t ) {
+  let capText = 'dataNormalization=';
+  capText += isPerCapita ? 'Per%201000%20pop.' : 'None';
+  const stateUrl = 'search/?dateInterval=3y&' + capText +
+    '&state=' + t.point.name;
+  const loc = location.protocol + '//' + location.host + location.pathname;
+  location.assign( loc + stateUrl );
+}
```
[the remark this answers is not in the document]
Comment (sephcoster) [reply cfgov/unprocessed/apps/ccdb-landing-map/js/TileMap.js:212]: Will follow-up with the customer for a post-deploy action. Thanks and good point.

File: cfgov/legacy/templates/complaint/complaint-landing.html [hunk 4]
```diff
@@ -77,27 +75,55 @@ <h2 class="h3">
...
       </div>
+
+      <div class="block block__flush-top">
+        <h2 class="h3">New! Map complaints by state</h3>
+        <p>
+          Our interactive map shows complaints submitted during the last three years. You can switch between the total complaints submitted or complaints submitted per 1,000 population (the total number of complaints in the geographic area per 1,000 persons living in that area, based on the 2017 American Community Survey (ACS) data).</p>
+
+        <p>
+          Selecting a state takes you to the Consumer Complaint Database where you can apply more filters or change the time period. 
+          <a href="search/">View interactive map</a>
```
[the remark this answers is not in the document]
Comment (sephcoster) [reply cfgov/legacy/templates/complaint/complaint-landing.html:86]: Will follow-up with the customer on this - thanks!

**Pull request merged**